include/clap/events.h

```diff
@@ -144,6 +144,12 @@ enum {
 // Well constructed plugins will search for voices and notes using
 // the entire tuple.
 //
+// In the case of note on events:
+// - The port, channel and key must be specified with a value >= 0
+// - A note-on event with a '-1' for Port Channel or Key is invalid and
+//   can be rejected or ignored by a plugin.
```

Comment:
It can also be rejected by the host then. The host can be consuming the plugin's note outupt.

Reply:
Will update!